shiny.html

```diff
@@ -7,64 +7,149 @@
 <meta charset="utf-8" />
 <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
 <meta name="generator" content="pandoc" />
-<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />
```

Comment:
Please discard all changes in the generated output. We'll compile the Rmd by ourselves.

Reply:
So it would be easier for me if you rejected this pull request and I do a new one.

Reply:
Done. I have deleted the fork and I have reforked. Please reject this pull request, I'm going to do the new one.